MetadataExtractor.Tests/MetadataExtractor.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.3</TargetFramework>
```

Comment:
The MetadataExtractor is targeting `netstandard1.3` and not `netstandard1.0`, this causes a lot of warning. The test should probably target the same TFM as the library.